src/content/reference/react-dom/hooks/useFormStatus.md

```diff
@@ -30,6 +30,7 @@ const { pending, data, method, action } = useFormStatus();
 The `useFormStatus` Hook provides status information of the last form submission.
 
 ```js {4},[[1, 5, "status.pending"]]
```

Comment:
then we should do this

```suggestion
```js {5},[[1, 6, "status.pending"]]
```

Reply:
Thank you!